### DISCUSSION TITLE
Why was interaction.channel removed

### DISCUSSION THREAD

**Discussion by Flo4604** [discordjs/discord.js#6068] [Q&A] [answered]
Title explains it.

**Comment by ImRodry** [comment, marked as answer]
It... wasn't?

**Reply by Flo4604** [reply]
my code likes to thinks so, TypeError: Cannot read property 'parent' of null
at Object.run (/home/tha/workspace/discordWeeb/src/commands/user/user.js:10:29)

the line is `interaction.channel.parent.name.toLowerCase() === 'staff'`

**Reply by Flo4604** [reply]
or am I missing a intent // partial

**Reply by ImRodry** [reply]
Null is not undefined. That happens likely because you ran that in a DM channel that wasn't cached

**Reply by Flo4604** [reply]
The channel is a normal guildChannel and not a dm could be that its not cached tho

**Reply by Flo4604** [reply]
Yeah, my bad had something to do with cache and I went back to using intents.all

**Reply by ImRodry** [reply]
Intents.all is no longer a thing, you need to actually specify intents

**Reply by Flo4604** [reply]
Yeah, I saw that with other updates so I just got my old package-lock and installed that version, As I will fix the setPermissions and intents and whatever else changed locally and not on my vps :p
